lib/data_update_scripts/20200519142908_re_index_feed_content_to_elasticsearch.rb

```diff
@@ -0,0 +1,28 @@
+module DataUpdateScripts
+  class ReIndexFeedContentToElasticsearch
+    def run
+      clear_existing_feed_documents
+
+      index_docs(Article.pluck(:id), "Article")
+      index_docs(PodcastEpisode.pluck(:id), "PodcastEpisode")
+      index_docs(Comment.pluck(:id), "Comment")
+    end
+
+    private
+
+    def clear_existing_feed_documents
```

Comment:
You definitely do not want to do this, none of the documents have bad data, we just want to update them so there is no need to clear them out.